be/src/util/coding.h

```diff
@@ -143,7 +143,7 @@ extern uint8_t* encode_varint64(uint8_t* dst, uint64_t value);
 inline uint8_t* encode_varint64(uint8_t* dst, uint64_t v) {
     static const unsigned int B = 128;
     while (v >= B) {
-        *(dst++) = (v & (B - 1)) | B;
+        *(dst++) = v | B;
```

Comment:
v is a uint64_t.
if v > 128, (v & (B -1)) | B not equal v | B

Reply:
(v & (B-1)) | B = (v | B) & ((B-1) | B ) = (v | B) & (11111111) = v | B
 
eg: 
129 & （128 - 1）  ｜ 128 = 1
129 ｜ 128 =  1

Reply:
(v | B) & (11111111) != v | B

If v = 1011111111
(v | B) = 1011111111
(v | B) &  (11111111) =  11111111

Reply:
v >>= 7;

In the loop, low seven bits of v shift right dst every time  and the eight bit is used to mark whether the number is continue to compression.   The low eight bits is assgined to *dst whitch is uint_8. 

In low eight bits,  (v | B) & (11111111) = 11111111.

Reply:
![image](https://user-images.githubusercontent.com/67053339/90461205-42999100-e138-11ea-980f-5f6530b88ccf.png)

This is the implementation in leveldb.